x-pack/packages/ml/query_utils/src/filter_based_default_query.test.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isFilterBasedDefaultQuery } from './filter_based_default_query';
+import { simpleQueryMock } from './__mocks__/simple_query';
+import { matchAllQuery } from './match_all_query';
+import { defaultSimpleQuery } from './simple_query';
+
+describe('isFilterBasedDefaultQuery', () => {
+  it('should identify filter based default queries', () => {
```

Comment:
I would separate `should identify filter-based default queries` and `should identify non-default queries`

Reply:
👍 updated in [6e6d393](https://github.com/elastic/kibana/pull/176534/commits/6e6d393dd1511f123717dc58ebfedeec3f8c88b2).